connaisseur/tests/test_mutate.py

```diff
@@ -594,7 +594,7 @@ def get_ad_request(path: str):
         (request_obj_daemonset, request_obj_output),
         (request_obj_statefulset, request_obj_output),
         (request_obj_job, request_obj_output),
-        (request_obj_tuftuf, None),
+        (request_obj_tuftuf, []),
```

Comment:
@phbelitz  I had to change this, because suddenly the test failed. Is it a problem?

Reply:
nope, seems fine.